credentials.example.json

```diff
@@ -69,6 +64,66 @@
       "accounts": { "mnemonic": "" },
       "chainId": 30,
       "url": "https://mainnet.sovryn.app/rpc"
+    },
+    "goerli": {
```

Comment:
Why move this down? Ethereum and testnets (goerli, sepolia) are supposed to be at the top.